File: jumping_on_the_clouds.js
"use strict"

const fs = require("fs")

process.stdin.resume()
process.stdin.setEncoding("utf-8")

let inputString = ""
let currentLine = 0

process.stdin.on("data", (inputStdin) => {
  inputString += inputStdin
})

process.stdin.on("end", (_) => {
  inputString = inputString
    .replace(/\s*$/, "")
    .split("\n")
    .map((str) => str.replace(/\s*$/, ""))

  main()
})

function readLine() {
  return inputString[currentLine++]
}

// Complete the jumpingOnClouds function below.
function jumpingOnClouds(c, s = 0) {
  if (c.length == 1) return s
  if (c.length > 2) if (c[2] == 0) return jumpingOnClouds(c.slice(2), s + 1)
  if (c[1] == 0) return jumpingOnClouds(c.slice(1), s + 1)
}

function main() {
  const ws = fs.createWriteStream(process.env.OUTPUT_PATH)

  const n = parseInt(readLine(), 10)

  const c = readLine()
    .split(" ")
    .map((cTemp) => parseInt(cTemp, 10))

  let result = jumpingOnClouds(c)

  ws.write(result + "\n")

  ws.end()
}

File: jumping_on_the_clouds.py
#!/bin/python3

import math
import os
import random
import re
import sys

# Complete the jumpingOnClouds function below.
def jumpingOnClouds(c, s=0):
    if len(c) == 1:
        return s
    if len(c) > 2:
        if c[2] == 0:
            return jumpingOnClouds(c[2:], s + 1)
    if c[1] == 0:
        return jumpingOnClouds(c[1:], s + 1)


if __name__ == '__main__':
    fptr = open(os.environ['OUTPUT_PATH'], 'w')

    n = int(input())

    c = list(map(int, input().rstrip().split()))

    result = jumpingOnClouds(c)

    fptr.write(str(result) + '\n')

    fptr.close()
